andela/dark-knight-rc
# #157911833 Vendor onboarding

#### What does this PR do?
Implement feature to allow vendor to effectively onboard onto the app

#### Description of Task to be completed?
1. Create plugin folder: vendorTour in imports/plugins/custom
2. Implement takeTour component to render the tour images using intro.js library
3. Upload tour images in cloudinary
4. Enable cloudinary url in policy.js to prevent content security policy violation error
5. npm install intro.js and intro.js-react
6. Import style for intro.js into takeTour.js component
7. Modify mainDropdown.js to dynamically render takeTour component when logged in user is a vendor/admin
8. Import Steps from intro.js-react and implement in mainDropdown file
9. Remove gitignore rule for imports/plugins/custom/*

#### How should this be manually tested?
1. On terminal (inside project folder), run command `reaction`
2. Go to localhost:3000 in your browser and click menu at the top right
3. Login to the app as admin, for "Tour App" button to appear in menu
4. Click "Tour App" button to be effectively onboarded as a vendor!

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#157911833](https://www.pivotaltracker.com/story/show/157911833)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-06-16 at 11 42 34 am" src="https://user-images.githubusercontent.com/29695968/41498013-672850ce-715a-11e8-8594-eafb58be5a17.png">
<img width="1440" alt="screen shot 2018-06-16 at 11 41 52 am" src="https://user-images.githubusercontent.com/29695968/41498017-71c6bba6-715a-11e8-99ad-e75be242f48a.png">
<img width="1440" alt="screen shot 2018-06-16 at 11 42 22 am" src="https://user-images.githubusercontent.com/29695968/41498019-7e626a72-715a-11e8-97ec-95f389a40845.png">
